examples/composed-auth/auth/authorizers.go

```diff
@@ -95,13 +95,18 @@ func HasRole(token string, scopes []string) (*models.Principal, error) {
 			isInScopes := false
 			claimedRoles := []string{}
 			for _, scope := range scopes {
+				isInScopes = false
 				for _, role := range claims.Roles {
 					if role == scope {
 						isInScopes = true
-						// we enrich the principal with all claimed roles within scope (hence: not breaking here)
+						// we enrich the principal with all claimed roles within scope
 						claimedRoles = append(claimedRoles, role)
+						break
 					}
 				}
+				if isInScopes == false {
+					break
+				}
```

Comment:
It looks like you want to achieve something else than the what I wanted to demonstrate here.
It might be interesting, although I can't clearly see what is the expected behavior here.
A good way to illustrate how to play with this differently could be to contribute a full example with the specifics of your use case explained.

Reply:
As I said above, I break the scope loop if the scope has not been found in the roles claimed.
The inScope will be false and the HasRole function will fail.
My purpose in the modification is to force all scopes to be present in the jet roles claimed to succeed the function.

Reply:
The intent documented with this function was "// We verify that the claimed role is **one of** the passed scopes".
I understand that you want a **all of** check. 
If this is the case, please add a function with this semantics and propose an additional path in the example that demonstrates this feature.

Reply:
Ok, I understand. Thanks you